include/odp/api/spec/ipsec.h

```diff
@@ -1146,7 +1142,7 @@ typedef enum odp_ipsec_status_id_t {
 	 *  This status event is generated only for outbound SAs in
 	 *  ODP_IPSEC_OP_MODE_INLINE mode.
 	 */
-	ODP_IPSEC_STATUS_WARN
+	ODP_IPSEC_STATUS_WARN = 0,
```

Comment:
The preceding comment says that this status event is generated only for outbound SAs in the inline mode, but I think that is not the intention anymore, at least not for the sa_disabled warning. Maybe the sentence should continue, e.g.: ...except in the case of sa_disabled warnings.

Reply:
We probably should just move this phrase to exact warnings description.